zulipterminal/cli/run.py

```diff
@@ -174,11 +174,14 @@ def main() -> None:
     except Exception as e:
         if args.debug:
             # A unexpected exception occurred, open the debugger in debug mode
-            import pudb
-            pudb.post_mortem()
 
-        sys.stdout.flush()
-        traceback.print_exc(file=sys.stderr)
+            sys.stdout.flush()
+            traceback.print_exc(file=sys.stderr)
+            run_debugger = input("Run Debugger? (y/n): ")
```

Comment:
I've not really used the debugger much; this now *only* shows the traceback if debug mode is enabled?

Showing the traceback was discussed as ok previously, I think? We can discuss this, but what do you think to leaving `-d` automatically go into the debugger, but tracebacks being logged to files (maybe with dates in their name)? We could add some text to request a description, which could be put in the top of the traceback file, and then ask for that file to be sent/posted to czo or some email that would post to a czo stream?

Reply:
Yup, this definitely seems like a thing to do. 👍

Reply:
To clarify, my thoughts would be to isolate the tracebacks, but offer the user a chance to write what they were just doing, as a way to annotate the issue - and that shouldn't depend on debug mode being enabled. I think it would be useful to keep `debug.log` as a general debugging tool, separate from crash-reporting?